misk/src/test/kotlin/misk/client/TypedHttpClientTest.kt

```diff
@@ -42,16 +44,25 @@ internal class TypedHttpClientTest {
   @Inject
   private lateinit var jetty: JettyService
 
-  private lateinit var client: ReturnADinosaur
+  private lateinit var clientInjector: Injector
```

Comment:
why inject the injector instead of just injecting the fields you need?

Reply:
Not injecting the injector. We're creating a custom one in the beforeEach that represents the client side injector. Injecting fields here would get them created with the wrong injector (and set of modules). This is the same as what was there before, just making it a member variable instead of a local one so that I can get the appropriate client.

Reply:
got it. thanks